order-service/src/main/resources/application-local.properties

```diff
@@ -0,0 +1,23 @@
+springfox.documentation.swagger.v2.path=/api-docs
+server.port=8080
+spring.jackson.date-format=com.anitechcs.orderservice.RFC3339DateFormat
+spring.jackson.serialization.WRITE_DATES_AS_TIMESTAMPS=false
+logging.level.io.swagger.models.parameters.AbstractSerializableParameter=error
+spring.jpa.properties.hibernate.globally_quoted_identifiers=true
+
+#H2 db configuration
+spring.datasource.url=jdbc:h2:mem:testdb
```

Comment:
Can we use the same DB name as Postgres?

Reply:
Okay. Will update. But this is just a file name where our values going to saved in file format